### DISCUSSION TITLE
Please integrate Builderius with Winden plugin (Tailwind for Wordpress)

### DISCUSSION THREAD

**Discussion by akcreation** [builderius/builderius#33] [Ideas]
Hi guys, I would very much like to be able to try combining Builderius with the Tailwind framework on Wordpress. In order to take advantage of the JIT CSS optimisation which works brilliantly on the 'Winden' plugin, it would be great if you would build an integration with that plugin.
Winden is developed by the guys at dplugins.com (and it also has it's own dedicated product website at winden.dev)
Many thanks,
Keith

**Comment by mrpsiho** [comment]
This is very good suggestion! The foundation task would be "Implementing classes manager". Once we have that - we would be ready to add this one.

**Comment by mrpsiho** [comment]
Builderius now has integration with WindPress PRO version.
